units/ssh-access.target

```diff
@@ -0,0 +1,12 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=SSH Access Available
+Documentation=man:systemd.special(7)
```

Comment:
Maybe, `RefuseManualStart=yes`?

Reply:
I think this should be OK to start manually, for example if you manually started sshd on the cmdline or weird cases like this. I think there's no strong reasons to prohibit this